s2a_options.go

```diff
@@ -78,6 +80,24 @@ const (
 	Spiffe
 )
 
+// S2AStream defines the stream that S2A must use to connect to the S2A server.
+type S2AStream interface {
+	Send(*s2av2pb.SessionReq) error
+	Recv() (*s2av2pb.SessionResp, error)
+}
+
+type S2AGrpcStream struct {
```

Comment:
Rather than offering a gRPC implementation, I'd prefer if this is the default when `GetS2AStream` is not populated.

Reply:
Done.